bitnami/discourse/Chart.yaml

```diff
@@ -35,4 +35,4 @@ sources:
   - https://github.com/bitnami/bitnami-docker-discourse
   - https://github.com/spinnaker
   - https://www.discourse.org/
-version: 7.0.16
+version: 7.0.166
```

Comment:
I think you accidentally added this change here instead of your fork, in order to trigger the tests.